src/config.ts

```diff
@@ -9,4 +9,5 @@ export const config = {
   algoliaIndexName: 'product',
   compareKeys: [],
   shopkitSrc: 'https://btn.moltin.com',
+  endpointURL: 'api.moltin.com',
```

Comment:
@petro97 are you sure this shouldn't be prefixed with the protocol? "https://api.moltin.com"